services/datamanager/builtin/builtin.go

```diff
@@ -739,14 +750,14 @@ func pollFilesystem(ctx context.Context, wg *sync.WaitGroup, captureDir string,
 		case <-ctx.Done():
 			return
 		case <-t.C:
-			logger.Info("Polling")
+			logger.Infof("Polling, value of n: %d", deleteEveryNth)
```

Comment:
Perhaps move this into debug? Since otherwise will continuously spam logs. And say something like "checking disk usage"

Reply:
Will move!

Reply:
Fixed in other pr.